doc/radosgw/account.rst

```diff
@@ -187,6 +187,44 @@ Alternatively, you may want to create a new account for each existing user. In
 that case, you may want to add the ``--account-root`` option to make each user
 the root user of their account.
 
+Migrating Notification Topics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Account topics are supported only when the ``notification_v2`` feature is enabled,
+as described in `Bucket Notifications`_ and `Supported Zone Features`_.
+
+1. ``Migration Impact``: When a non-account user is migrated to an account,
+the user loses access to its existing notification topics via the SNS Topic API.
+However, these topics remain functional, and bucket notifications will continue
+to be delivered as expected.
```

Comment:
maybe add a note that they are still accessible by the admin API?

Reply:
Added some more clarification around this.